app/page.tsx

```diff
@@ -1,7 +1,37 @@
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+
 const Home = () => {
   return (
-    <main className="flex flex-col p-6 sm:p-12 flex-1 text-neutral-900">
-      <section id="about">
+    <main className="flex flex-col p-6 sm:p-12 flex-1">
+      <section className="min-h-[calc(100vh-80px)] sm:min-h-[calc(100vh-104px)] flex flex-col justify-center sm:pb-24 md:pl-6">
+        <h1 className="text-4xl sm:text-5xl lg:text-6xl text-center text-neutral-800 sm:text-left pb-8">
```

Comment:
I am conflicted on this one. Technically a webpage should only ever have one `h1` tag and with my quick merge conflict fixes, I have introduced two...

I am wondering since this is the company name if it would be more appropriate to have it be the image of our logo rather than an `h1` text element?